client/asset/dcr/dcr.go

```diff
@@ -6525,7 +6542,51 @@ func (dcr *ExchangeWallet) TicketPage(scanStart int32, n, skipN int) ([]*asset.T
 	return pager.TicketPage(dcr.ctx, scanStart, n, skipN)
 }
 
-func (dcr *ExchangeWallet) broadcastTx(signedTx *wire.MsgTx) (*chainhash.Hash, error) {
+func (dcr *ExchangeWallet) broadcastTx(signedTx *wire.MsgTx, feeRate uint64) (*chainhash.Hash, error) {
+	// Hard limit: Validate transaction size before broadcasting to prevent
+	// transactions from getting stuck in mempool.
+	// Use MaxStandardTxSize (100KB) not chainParams.MaxTxSize (393KB) because
+	// the mempool enforces a 100KB policy limit for transaction relay.
+	// Transactions larger than 100KB will be rejected by the mempool even though
+	// they would be valid according to consensus rules.
+	txSize := signedTx.SerializeSize()
+	maxSize := uint64(dexdcr.MaxStandardTxSize)
```

Comment:
This should be the network specific one.

Reply:
please read comment above

Reply:
> If the tx is larger than MaxTxSize (393,216 bytes) it is INVALID by consensus and will be rejected by the network, the block containing such tx will be rejected.

Then our tx is stuck isnt it? It will spin thinking its in mempool forever. We will send and it will not be included in a block. Will not get confirmations. Will stay spinning in the UI. That is the issue we are trying to fix.